packages/component-library/stories/ScatterPlotMap.notes.md

```diff
@@ -0,0 +1,39 @@
+# ScatterPlot Map Component
+
+## Standard
+
+The Standard story shows the standard usage of the ScatterPlot Map Component for the CIVIC platform. ScatterPlot Map includes the standard styling.
+
+These properties can be set in the standard story:
+
+- `getFillColor`
+- `radiusScale`
+- `opacity`
+- `data`
+
+## Custom
+
+The Custom story shows all the possible properties that can be passed to the ScatterPlot Map Component for customization.
+
+- `data` : expects an array of objects.
+- `getPosition` : expects a function that returns the coordinates of each object in data array.
+- `opacity` : expects a number.
```

Comment:
It would be good to note here that the number to set the opacity should be between 0 and 1.